core/trino-main/src/test/java/io/trino/metadata/TestQualifiedTablePrefix.java

```diff
@@ -62,15 +62,10 @@ public void testTable()
     }
 
     @Test
-    public void testBadTable()
+    public void testNullSchema()
     {
-        try {
-            new QualifiedTablePrefix("catalog", null, "table");
-            fail();
-        }
-        catch (RuntimeException e) {
-            // ok
-        }
+        assertThatThrownBy(() -> new QualifiedTablePrefix("catalog", null, "table"))
+                .hasMessage("schemaName is null");
```

Comment:
validate exception class

Reply:
previously the test was expecting any RTE, but indeed, i should add this